dist/index.js

```diff
@@ -29819,40 +29819,28 @@ async function getFullVersionFromStarknetFoundry() {
   return match[1];
 }
 
-async function determineVersion(version, toolVersionsPath, repo) {
-  version = version?.trim();
```

Comment:
note: It's trimmed by default
https://github.com/actions/toolkit/blob/683703c1149439530dcee7b8c5dbbfeec4104368/packages/core/src/core.ts#L151-L162